### PR TITLE
OrtResult stuff

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/1300)

### CODE REVIEW

File: model/src/main/kotlin/OrtResult.kt [hunk 1]
```diff
@@ -62,8 +62,8 @@ data class OrtResult(
         val data: CustomData = emptyMap()
 ) {
     /**
-     * Return the concluded licenses associated to their package. If [omitExcluded] is set to true, excluded packages
-     * are omitted from the result.
+     * Return the concluded licenses for each package. If [omitExcluded] is set to true, excluded packages are omitted
```
Comment (mnonnenmacher) [inline model/src/main/kotlin/OrtResult.kt]: Commit message nits:
* woring -> wording
* not -> note

File: model/src/main/kotlin/OrtResult.kt [hunk 2]
```diff
@@ -157,6 +157,33 @@ data class OrtResult(
         return analyzerErrors.zipWithDefault(scannerErrors, emptySet()) { left, right -> left + right }
     }
 
+    /**
+     * Return the set of all project or package identifiers in the result, optionally [including those of sub-projects]
+     * [includeSubProjects].
+     */
```
Comment (mnonnenmacher) [inline model/src/main/kotlin/OrtResult.kt]: What is the use case for excluding project which other projects depend on?

Comment (sschuberth) [reply model/src/main/kotlin/OrtResult.kt]: The use-case is that for sub-projects we may not want to perform certain checks, like verifying that they have a detected license.

File: model/src/main/kotlin/OrtResult.kt (hunk 1 above)
Comment (sschuberth) [reply model/src/main/kotlin/OrtResult.kt]: Done.

**Pull request merged**